src/Controls/tests/TestCases.HostApp/Issues/Issue24147.cs

```diff
@@ -0,0 +1,20 @@
+namespace Maui.Controls.Sample.Issues
+{
+	[Issue(IssueTracker.Github, 24174, "DatePicker does not leak", PlatformAffected.All)]
+	public class Issue24174 : NavigationPage
+	{
+		public Issue24174() 
+		{
+            this.RunMemoryTest(() =>
```

Comment:
Can we look at generalizing this test a bit more?

I think we're going to start moving a few of these from device tests to here and it'd probably be good not to have them spread out.

I'm thinking

- rename this to MemoryTests.cs
- Add an entry box and a button
- the test can now uses testcases to enter text and click the button for which test to run